provisioning/device/tests/CertificateInstallerTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using FluentAssertions;
+using Microsoft.Azure.Devices.Provisioning.Client.Utilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.UnitTests
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class CertificateInstallerTests
+    {
+        [TestMethod]
+        public void CertificateInstaller_EnsureChainIsInstalled_Works()
+        {
+            // arrange
```

Comment:
leave an extra line when we have a code block below, since we want to follow with the same standard here and there